### PR TITLE
Add tooltip rendering to all panes

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Add tooltip rendering to all panes

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/blockout/Pane.java [hunk 1]
```diff
@@ -344,12 +352,33 @@ public final void drawLast(final int mx, final int my)
         if (visible)
         {
             drawSelfLast(mx, my);
+
+            if (isHovered && toolTipLines != null && !toolTipLines.isEmpty())
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/blockout/Pane.java]: just initialize toolTipLines with an empty list so we don't have to do both checks here

File: src/main/java/com/ldtteam/blockout/Pane.java [hunk 2]
```diff
@@ -805,4 +831,14 @@ public boolean onMouseDrag(final double mx, final double my, final int speed, fi
     {
         return false;
     }
+
+    /**
+     * Sets the tooltip to render on hovering this element
+     *
+     * @param lines the lines to display
+     */
+    public void setHoverToolTip(final List<String> lines)
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/blockout/Pane.java:842]: There should probably be a way to set it via pane params too.

Comment (someaddons) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: nah this is easier :D

**Review by Nightenom — COMMENTED** [review]
Probably need hooks for translation similar to what label element is using. 
Also I'm 80% sure we don't need to copy the forge class
Edit: I can finish this at Tuesday if you are not in hurry

File: src/main/java/com/ldtteam/blockout/Pane.java (hunk 2 above)
Comment (Nightenom) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: Agree with Ray honestly

Comment (Raycoms) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: Oh, I mean we should have both in this regards.

**Comment by Raycoms** [comment]
We need to copy the forge class because the forge class uses itemStacks and can't do without. We have to remove those.

File: src/main/java/com/ldtteam/blockout/Pane.java (hunk 2 above)
Comment (someaddons) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: I think we only should pass essential and required information as constructor parameters, not optional ones. Else we run into a pleathora of default constructors with less parameters for each and every class extending pane

Comment (Raycoms) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: not in the constructor. To the paneparams object.

Comment (someaddons) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: should've said so to start with! lately you're very sparse with details xD

Comment (Raycoms) [reply src/main/java/com/ldtteam/blockout/Pane.java:842]: Marie Kondo

**Pull request merged**